src/CONST.ts

```diff
@@ -2543,6 +2543,7 @@ const CONST = {
             MONTHLY: 'monthly',
             FIXED: 'fixed',
         },
+        LIMIT_VALUE: 20000000,
```

Comment:
@tgolen friendly bump here

Reply:
I think we should make it consistent, yeah. Do you need me to get the backend value?

Reply:
OK, I looked at the backend, there is no server validation done on the actual limit value, so I guess it would make sense to set this to `21,474,836.47`

Reply:
@mkhutornyi i updated, please check again